### ISSUE TITLE
GGMAX Enhancement - Increase in the number of screens/levels required

### ISSUE THREAD

**Issue by extremestrategy** [TheGameCreators/GameGuruRepo#5715]
While working on my game I came across a problem which is the Storyboard can only have a max of 50 levels and screens, this is a problem because my game is mission based and each mission has to have a map selector, mission selector, loading screen, then the mission itself, if the amount of screens/levels could be increased I would very much appreciate it very much.

![Screenshot 2024-09-08 203519](https://github.com/user-attachments/assets/4b8c1a6b-9e3e-405f-a6e6-d47affbeb6c9)

**Comment by MonkeyFrogStudio** [comment]
Yeah, technically, I think there should be no real limits (or the limit set so high, there's no way anyone would ever reach it). However, the storyboard, imo, needs a bit of redesign in order to be useable/readable with all that's going on with very large games. For example, the HUD stuff (crafting, basic UI, etc.) should be it's own, separate thing, leaving the storyboard for only levels, loading screens, etc.

**Comment by Necrym59** [comment]
Perhaps it should branch the storyboard into three tabbed sections, one for **'system menus'** one for **'game screens'** and one for **'huds'**

**Comment by MonkeyFrogStudio** [comment]
That would work. Anything to unclutter the space for more complex games.

**Comment by Necrym59** [comment]
Plus while where at it, can we have one common loading screen for multiple levels that would cut down the number of screens by a big amount and would resolve some of this issue as well.

**Comment by extremestrategy** [comment]
That would be an excellent idea Necrym.

**Comment by C4ever08** [comment]
Yes I agree one loading screen would be better but then it limits the loading screen images per level. I think hud, level and loading screens should be separate. I'm at Max screens atm so It does stop me moving forward for now but It'll be sorted.
